lib/adapters/test-result/playwright.ts

```diff
@@ -127,7 +146,8 @@ const getImageData = (attachment: PlaywrightAttachment | undefined): ImageFile |
 
     return {
         path: attachment.path as string,
-        size: _.pick(sizeOf(attachment.path as string), ['height', 'width']) as ImageSize
+        size: !attachment.size ? _.pick(sizeOf(attachment.path as string), ['height', 'width']) as ImageSize : attachment.size,
```

Comment:
When image is updated I already have size value and don't want to recalculate it again